lib/rex/post/mssql/ui/console/command_dispatcher.rb

```diff
@@ -79,32 +25,13 @@ def docs_dir
             ::File.join(super, 'mssql_session')
           end
 
-          #
-          # Returns true if the client has a framework object.
-          #
-          # Used for firing framework session events
-          #
-          # @return [TrueClass, FalseClass]
-          def msf_loaded?
-            return @msf_loaded unless @msf_loaded.nil?
-
-            # if we get here we must not have initialized yet
-
-            @msf_loaded = !session.framework.nil?
-            @msf_loaded
-          end
-
           #
           # Log that an error occurred.
           #
           # @param [Object] msg
           # @return [Object]
           def log_error(msg)
-            print_error(msg)
-
-            elog(msg, 'mssql')
-
-            dlog("Call stack:\n#{$ERROR_POSITION.join("\n")}", 'mssql')
+            super(msg, protocol: 'mssql')
```

Comment:
Instead of overriding this, can we just implement the call to self.type in the `super` implementation? 🤔